### PR TITLE
Feature/implement client states - continue with other socket responses

### PR DESCRIPTION
**Pull request by mrvary**
Continuation of #81

### CODE REVIEW

File: src/assets/mock-data/match.mock.data.js [hunk 1]
```diff
@@ -32,4 +32,37 @@ const mockedMatch = {
   freeTicket: false
 };
 
-module.exports = { mockedMatch };
+const mockedMatchRunning = {
+  id: 1,
+  player1: {
```
Comment (Adentalis) [inline src/assets/mock-data/match.mock.data.js]: player1: player.id ,not the whole player

File: src/assets/mock-data/match.mock.data.js [hunk 2]
```diff
@@ -32,4 +32,37 @@ const mockedMatch = {
...
+    active: true,
+    hasFreeTicket: false
+  },
+  sets: [
+    { player1: 11, player2: 8 },
+    { player1: 8, player2: 11 },
+    { player1: 10, player2: 12 },
+    { player1: 15, player2: 13 }
+  ],
+  freeTicket: false
```
Comment (Adentalis) [inline src/assets/mock-data/match.mock.data.js]: we don't use it anymore

File: src/assets/mock-data/match.mock.data.js (hunk 1 above)
Comment (mrvary) [reply src/assets/mock-data/match.mock.data.js]: I think in the frontend it is the whole player that gets send. @Ceecker51 ?

File: src/assets/mock-data/match.mock.data.js (hunk 2 above)
Comment (mrvary) [reply src/assets/mock-data/match.mock.data.js]: so it's only a free ticket player then?

Comment (Adentalis) [reply src/assets/mock-data/match.mock.data.js]: yes
Even though there should be never a Freeticket game/player on the client

File: src/assets/mock-data/match.mock.data.js (hunk 1 above)
Comment (Adentalis) [reply src/assets/mock-data/match.mock.data.js]: it can be send this way as well of course :)

File: src/assets/mock-data/match.mock.data.js (hunk 2 above)
Comment (mrvary) [reply src/assets/mock-data/match.mock.data.js]: good point!

File: src/assets/mock-data/match.mock.data.js (hunk 1 above)
Comment (Ceecker51) [reply src/assets/mock-data/match.mock.data.js]: https://confluence.student.fiw.fhws.de:8443/display/PAWS1920TTRace/Matchmaker

**Pull request merged**